code/game/turfs/simulated/floor/asteroid_floors.dm

```diff
@@ -314,7 +375,11 @@ GLOBAL_LIST_INIT(megafauna_spawn_list, list(/mob/living/simple_animal/hostile/me
 #undef SPAWN_BUBBLEGUM
 
 /turf/simulated/floor/plating/asteroid/airless/cave/proc/SpawnFlora(turf/T)
-	if(prob(12))
+	var/floraprob = 12
+	switch(SSmapping.cave_theme)
+		if(BLOCKED_BURROWS)
+			floraprob = 30 //Lots of folliage, lots of blockage
```

Comment:
switch with one thing in it 😠

Reply:
Do we want to make this switches so it is modular and easy to change, or do we want to remove switches so further expansions to generation have to add them back?